src/www/ui_components/server_connection_viz/index.ts

```diff
@@ -0,0 +1 @@
+export * from './server_connection_viz';
```

Comment:
Why do you need this file? Can't the content of the server_connection_viz.ts be here instead?

Reply:
i could, but for it to live in the same folder i'd have to import like `'./server_connection_viz/server_connection_viz` - index lets me define this as a module and thus shortcut to the main component

Reply:
https://www.reddit.com/r/javascript/comments/7rv221/to_indexjs_or_not_to_indexjs/

Reply:
yeah, i guess so. it's just harder to search for the `index` file

Reply:
also it's nice to have this in the tab:
![Screen Shot 2022-02-04 at 12 01 17](https://user-images.githubusercontent.com/3759828/152571166-838d5fa6-556b-466d-bcc6-645010b0ffe2.png)

Reply:
Why do you need to name this `index.ts` though? Just use `server_connection_viz.ts` directly.
This export makes the code harder to understand. It's very unclear what the intent is. Let's remove this indirection.

Reply:
if i don't use a standard index file that then we have to import it this way - `./server_connection_viz/server_connection_viz` instead of just `./server_conneciton_viz`

Reply:
https://nodejs.org/api/modules.html#modules_folders_as_modules

Reply:
Let's delete this file and rename server_connection_viz.ts to index.ts.
It's a lot clearer, and we should avoid the name stuttering.

And rename server_connection_viz.stories.ts to stories.ts.

Then all components can have the same structure. Perhaps we can add tests too in the future.

Reply:
yeah, sure